scripts/test_updates_pg13.sh

```diff
@@ -3,6 +3,7 @@
 set -e
 
 SCRIPT_DIR=$(dirname $0)
+export PG_VERSION=13
```

Comment:
If you set this unconditionally here you should remove PG_VERSION env from caller in .github/workflows/update-test.yaml. Also this is unrelated to your change so i'd prefer to see this in separate commit.

Reply:
I don't think it make sense to have a script named `xyz_pg13.sh` and  then run it with, e.g., PG12. So, yes, we should probably make this unconditional or create a generic script that can run any postgres version. I'll leave it for a separate PR, though.